tools/HelixQAC/README.md

```diff
@@ -0,0 +1,71 @@
+# Using Helix QAC 2021.1 on iceoryx
+
+
+## prerequisites
+
+This requires that you have :
+
+ - iceoryx that can be built
+ - Helix QAC 2021.1
+ - ASCM 2.4.0
+ - the latest CCT Generator ( can be found [here](http://products.prqa.com/Common/CCT_Generator/) )
+
+## Generating a CCT
+
+This step depends on the compiler you use, but in general the command is along the lines of 
+
+    CCT_Generator.sh -c <compiler command>
+
+with the compiler command being a command used to build an iceoryx file. If you do not know how your project is being built, check build logs.
+
+an example command if you use GCC (command taked from the build log):
+
+    CCT_Generator.sh -c /usr/bin/c++  -Diceoryx_utils_EXPORTS -I/home/perforce/iceoryx/iceoryx_utils/include -I/home/perforce/iceoryx/iceoryx_utils/platform/linux/include  -O3 -DNDEBUG -fPIC   -W -Wall -Wextra -Wuninitialized -Wpedantic -Wstrict-aliasing -Wcast-align -Wno-noexcept-type -Wconversion -std=gnu++14 -o CMakeFiles/iceoryx_utils.dir/source/posix_wrapper/unix_domain_socket.cpp.o -c /home/perforce/iceoryx/iceoryx_utils/source/posix_wrapper/unix_domain_socket.cpp
+the command can be simplified to it's necessary items (we can remove all warning-related flags, source file, output, include and defines for example), but the Generator can take care of the cleaning.
+
+If you have issues using the CCT_Generator, please check the PDF manual supplied with it. Some compilers also require some environment being set (for licensing for example) so make sure that the compiler can be invoked in such a way.
+
+Once you generated the CCT, take note of the CCT's name as we will need it later :
+
+    Writing /home/perforce/.config/Perforce/Helix-QAC-2021.1/config/cct/GNU_GCC-c++_9.3.0-x86_64-linux-gnu-C++-c++14.cct
+
+
+## Creating the Helix QAC project
+
+we are going to create an empty Helix QAC project in the iceoryx folder, using default settings and rulesets, but with the previously generated CCT. e.g. :
+
+    qacli admin --qaf-project-config -P . --cct GNU_GCC-c++_9.3.0-x86_64-linux-gnu-C++-c++14.cct --rcf default-en_US.rcf --acf default.acf
+
+this will create a *prqaproject.xml* file and *prqa* folder.
+
+note that since iceoryx is only C++, we only provide a C++ CCT.
+
+## adding iceory files to the Helix QAC Project
+this can be done in many ways, but we will go with the most straightforward one : process monitor.
+supposing you build with ` cmake --build build`, you will run the command
```

Comment:
May mention that you have to create all cmake files before running this command, e.g. by executing `cmake -Bbuild -Hiceoryx_meta`

Reply:
thank you. I thought putting "iceoryx that builds" in the requirements was enough. With the latest commit made things more explicit, let me know if this is still unsatisfactory.